packages/plugin-flag/test/integration/spec/flag.js

```diff
@@ -90,7 +90,7 @@ describe(`plugin-flag`, function() {
 
     describe(`#list()`, () => {
       it(`fetches the flag list`, () => {
-        return spock.spark.flag.list()
+        return spock.spark.flag.get()
```

Comment:
get should retrieve a single item, list should retrieve all (or a filter subset thereof)